server.js

```diff
@@ -7479,7 +7480,16 @@ cache({
     }
     var url = encodeURI(decodeURIComponent(query.url || query.uri));
 
-    request(url, (err, res, data) => {
+    if (type === 'json') {
+      requestOptions = {
+        headers: {
+          Accept: 'application/json'
+        },
+        json: true
+      };
```

Comment:
For consistency, should set `Accept: application/xml` and `Accept: text/xml` for xml?